homeassistant/components/keyboard_remote.py

```diff
@@ -67,22 +47,18 @@
 def setup(hass, config):
     """Setup keyboard_remote."""
     config = config.get(DOMAIN)
-    device_descriptor = config.get(DEVICE_DESCRIPTOR)
-    if not device_descriptor:
-        id_folder = '/dev/input/'
-        _LOGGER.error(
-            'A device_descriptor must be defined. '
-            'Possible descriptors are %s:\n%s',
-            id_folder, os.listdir(id_folder)
-        )
-        return
 
-    key_value = KEY_VALUE.get(config.get(TYPE, 'key_up'))
+    if not config.get(DEVICE_DESCRIPTOR) and\
+       not config.get(DEVICE_NAME):
+        _LOGGER.debug(
+            'Error: No device_descriptor\
```

Comment:
Just one more, I missed this last time: Remove "Error: " and make this function `_LOGGER.error`.